examples/vitalam-app/charts/vitalam-frontend/values.yaml

```diff
@@ -0,0 +1,35 @@
+##############################################################################################
+#  This Chart is a fork from https://github.com/digicatapult/vitalam-ipfs
+#  Please update if needed
+##############################################################################################
+
```

Comment:
value file should not have hardcoded values - should be generated by the helmtemplate job using varibles passed in via the ansible role

Reply:
Also, the copyright info seems to be wrong. If you are not taking the chart from that path you specified, there is no need to add attribution